Gemfile

```diff
@@ -174,12 +174,16 @@ end
 group :production do
   # sidekiq enterprise requires a license key to download but is only required in production.
   # for local dev environments, regular sidekiq works fine
-  unless (Bundler::Settings.new['enterprise.contribsys.com'].nil? ||
-          Bundler::Settings.new['enterprise.contribsys.com']&.empty?) &&
+  unless (Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com'].nil? ||
+          Bundler::Settings.new(Bundler.app_config_path)['enterprise.contribsys.com']&.empty?) &&
          ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', '').empty?
     source 'https://enterprise.contribsys.com/' do
       gem 'sidekiq-ent'
       gem 'sidekiq-pro'
     end
+  else
+    puts "No credentials found to install Sidekiq Enterprise. This is fine for local development but you may not check in this Gemfile.lock."
+    sleep 3
```

Comment:
Is this just so the message lingers in front of the user?

Reply:
Yeah -- otherwise its at the TOP of the list of gems and scrolls off screen. Don't love it, can dump it all together.